docs/README.md

```diff
@@ -0,0 +1,10 @@
+# Rainbow Bridge
+
+This documentation is about design, usage, maintenance, and testing of the NEAR-ETH bridge.
+
+* [Overview of the components](./components.md)
```

Comment:
I was looking for docs and hit a broken link here.
This file doesn't seem to exist anywhere in the repo?

Reply:
We forgot to add it. Thanks for noticing.